packages/plugin-infra/src/web3/useTransactions.ts

```diff
@@ -12,6 +12,6 @@ export function useTransactions<S extends 'all' | void = void, T extends Network
     const hub = useWeb3Hub(pluginID, options)
 
     return useAsyncRetry<Array<Web3Helper.TransactionScope<S, T>> | undefined>(async () => {
-        return hub?.getTransactions(chainId, account)
+        return hub?.getTransactions(options?.chainId ?? chainId, account)
```

Comment:
What about the `account`? Does it necessary?

```suggestion
        return hub?.getTransactions(options?.chainId ?? chainId, options?.account ?? account)
```